bot/resources/tags/traceback.md

```diff
@@ -1,18 +1,16 @@
 Please provide the full traceback for your exception in order to help us identify your issue.
+While the last line of the error message tells us what kind of error you got,
+the full traceback will tell us which line, and other critical information to solve your problem.
+Please avoid screenshots so we can copy and paste parts of the message.
 
 A full traceback could look like:
 ```py
 Traceback (most recent call last):
-    File "tiny", line 3, in
-        do_something()
+    File "my_file.py", line 3, in
+        add_three("6")
     File "tiny", line 2, in do_something
```

Comment:
```suggestion
    File "my_file.py", line 2, in add_three
```

Reply:
Thank you for pointing this out; I have fixed it.